doc/source/release/1.17.0-notes.rst

```diff
@@ -248,6 +248,19 @@ New features
   `scipy.stats.ks_1samp`, `scipy.stats.levene`, and `scipy.stats.mood`.
   Typically, this improves performance with multidimensional (batch) input.
 - The critical value tables of `scipy.stats.anderson` have been updated.
+- A new ``method`` parameter of `scipy.stats.anderson` allows the user
+  to compute p-values by interpolating between tabulated values or using Monte
+  Carlo simulation. The ``method`` parameter must be passed explicitly
+  to add a ``pvalue`` attribute to the result object and avoid a warning
+  about the upcoming removal of ``critical_value``, ``significance_level``,
+  and ``fit_result`` attributes.
+- A new ``variant`` parameter of `scipy.stats.anderson_ksamp` allows the user
+  to select between three different variants of the statistic, superseding the
+  ``midrank`` parameter which allowed toggling beteen two. The new ``'continuous'``
+  variant is equivalent to ``'discrete'`` when there are no ties in the sample, but
+  the calculation is faster. The ``variant`` parameter must be passed explicitly to
+  avoid a warning about the deprecation of the ``midrank`` attribute and the upcoming
+  removal of ``critical_values`` from the result object.
```

Comment:
This was already mentioned on line 351-353:

> - For `scipy.stats.anderson_ksamp`, the ``midrank`` parameter is deprecated
  and the new ``variant`` parameter should be preferred. This also means that
  the presence of the ``critical_values`` return array is deprecated.

But a bit of repetition wouldn't hurt I suppose